prbt_hardware_support/spec.dox

```diff
@@ -97,14 +97,9 @@
  *  - The system provides a mechanism which allows to determine if
  *    a braketest is requested by the safety controller.
  *
- *  - The mechanism continues to return the last received value,
```

Comment:
I am pretty sure, this also holds

Reply:
No because the condition changed. The way the spec is now defines everything.

Reply:
and it will not return the last received value in specific cases?